components/common/IdentityForm.vue

```diff
@@ -227,6 +251,26 @@ const {
 const isConfirmModalActive = ref(false)
 const isLoaderModalVisible = ref(false)
 const transactionValue = ref('')
+const menus = availablePrefixes().filter(
+  (menu) =>
+    menu.value !== 'movr' &&
+    menu.value !== 'glmr' &&
+    menu.value !== 'ahk' &&
+    menu.value !== 'ahp',
+)
+const chainByPrefix = computed(() =>
+  menus.find((menu) => menu.value === urlPrefix.value),
+)
+const selectChain = ref(chainByPrefix.value?.value || menus[0].value)
+watch(urlPrefix, (value) => {
+  selectChain.value = value
+})
+const currentChain = computed(() => selectChain.value as Prefix)
+watch(currentChain, () => {
+  if (currentChain.value !== urlPrefix.value) {
+    setUrlPrefix(currentChain.value as Prefix)
+  }
+})
```

Comment:
I'll put the computed & watcher together

Reply:
> I'll put the computed & watcher together

how

Reply:
> I'll put the computed & watcher together

you mean for currentChain

Reply:
obviously you can but in my opinion I don't think making changes in this line of code will make a any difference regardless of this you can check other select chain button on create page etc they work similar and there is no any major difference

Reply:
better reading